source/hardware/controllers/reloop_digital_jockey_2_interface_edition.rst

```diff
@@ -0,0 +1,58 @@
+.. _reloop_digital_jockey_2_interface_edition:
+
+Reloop Digital Jockey 2 Interface Edition
+=========================================
+
+The *Reloop Digital Jockey 2 Interface Edition* is a 2 deck USB MIDI controller designed to work with Traktor software. Compared to *Controller Edition* the *Interface Edition* is additionally equipped with a high quality 4-channel USB audio interface and touch-sensitive jog wheels. The microphone input is available to the computer for recording and broadcasting. The controller can be powered only from the USB bus, however the included external power supply can serve as the power source too. Using a separate power supply has no impact on the output level or LED brightness.
+
+-  `Manufacturer’s product page <https://www.reloop.com/reloop-digital-jockey-2-ie>`__
+-  `Forum thread <https://mixxx.discourse.group/t/reloop-digital-jockey-2-mapping-by-dj-ak/23971>`__
+
+.. versionadded:: 2.3
+
+Compatibility
+-------------
+
+The mapping for *Reloop Digital Jockey 2 Interface Edition* has originally been developed under Debian 8 Jessie with *Mixxx v2.1*. Lately the mapping has been moved to *Mixxx v2.3* and tested under Kubuntu 20.04. No special device drivers are required for GNU/Linux systems.
+
+Windows build has not been tested.
+
+Mixxx Sound Hardware Preferences
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+.. table::
+    :widths: 100 100
+
+    ========================  =======================
+    Output                    Channels
+    ========================  =======================
+    :guilabel:`Master`        Channel 1 - 2
+    :guilabel:`Headphones`    Channel 3 - 4
+    ========================  =======================
+
+Mapping
+-------
+
+The mapping is straightforward and follows the layout of the controller. The deviations are documented below.
+
+Decks
+~~~~~
+
+Although the mapping provided to Traktor users by the manufacturer has a feature of extending the controller to handle 4 decks using a :hwlabel:`SHIFT` button, this particular mapping handles the basic 2 decks setup. :hwlabel:`SHIFT` button has been mapped, however it handles the rest of the functionality.
+
```

Comment:
Irrelevant, can be removed. The introduction already says this is a 2-deck controller.

Reply:
The introduction describes hardware characteristics as there are 2 physical decks.
I think it is relevant to mention this, as Traktor mapping extends it to 4 decks. This can dispel user confusion when switching from Traktor to Mixxx.

Reply:
Traktor **Pro** 2:
> Reloop Digital Jockey Overlay:
Traktor Pro 2 overlay for Digital Jockey
Incl. new 4-deck assignment
Support for sample deck control
Fits all Digital Jockey gnerations
Made of sturdy PP foil

It is sufficient to say it can handle 2 decks in Mixxx.
The rest is irrelevant and potentially confusing, even more so the detailed description how it is handled in Traktor. Less is more. Consider that documentation is not translated and it should be "to the point" for users not speaking english.

Reply:
Fixed.